mixer/pkg/runtime/dispatcher/dispatcher_test.go

```diff
@@ -751,13 +786,16 @@ func TestDispatcher(t *testing.T) {
 
 			var err error
 			switch tst.variety {
-			case tpb.TEMPLATE_VARIETY_CHECK:
-				cres, e := dispatcher.Check(context.TODO(), bag)
+			case tpb.TEMPLATE_VARIETY_CHECK, tpb.TEMPLATE_VARIETY_CHECK_WITH_OUTPUT:
+				cres, dir, e := dispatcher.Check(context.TODO(), bag)
 
 				if e == nil {
 					if !reflect.DeepEqual(&cres, &tst.expectedCheckResult) {
 						tt.Fatalf("check result mismatch: '%v' != '%v'", cres, tst.expectedCheckResult)
 					}
+					if !reflect.DeepEqual(&dir, &tst.expectedCheckDirective) {
```

Comment:
worth using messagediff here (to generate more consumable diffs in the `Fatalf`s) ?